.tekton/push.yaml

```diff
@@ -137,7 +139,7 @@ spec:
                   [[ -f "$f" ]] && list+=("$f")
                 done
 
-                .tekton/scripts/build-acceptable-bundles.sh "${list[@]}"
+                hack/build-acceptable-bundles.sh "${list[@]}"
```

Comment:
Why move this file? I think it makes sense to live under `.tekton/scripts` since it's meant to be used during the CI of this repo. `hack/push-and-tag.sh` should probably go there as well.

Reply:
I moved it because `build-and-push.sh` is under hack and it just made it easier if all three were in the same location, since they all create and work with files as input. I can move them under `.tekton` if you feel strongly about it.